packages/actions-global/tsconfig.json

```diff
@@ -1,5 +1,4 @@
 {
   "extends": "../../tsconfig.json",
-  "files": ["./src/index.ts"],
-  "include": ["./src/globals.d.ts"]
```

Comment:
Those have been now moved to `<root>/@types`, as I figured out the correct setup of `tsconfig.json`